### ISSUE TITLE
Mistakes in `bottle-song` example solution typespecs

### ISSUE THREAD

**Issue by antoine-duchenet** [exercism/elixir#1371]
In the typespecs of `.meta/example.ex`, the arguments of `quantifier/1` and `bottle/1` should be `non_neg_integer`, not the current `pos_integer`, since it accepts `0` :

```elixir
defmodule BottleSong do
  ...
  @spec quantifier(non_neg_integer) :: String.t()
  defp quantifier(n), do: Map.get(@quantifiers, n)

  @spec bottle(non_neg_integer) :: String.t()
  defp bottle(1), do: "bottle"
  defp bottle(_), do: "bottles"
end
```

**Comment by jiegillet** [comment]
You're right, I missed this in the review. Do you want to open a PR?

**Comment by jiegillet** [comment]
Solved by #1375
